### PR TITLE
Add what's new section

### PR DESCRIPTION
**Pull request by tristanlatr**

### CODE REVIEW

**Comment by tristanlatr** [comment]
Hi @2ynn,

Could you very quickly check this new section that talk about the new improvements of saskatoon 2022 ?

Thanks a lot,

File: docs/source/index.rst [hunk 1]
```diff
@@ -33,8 +25,70 @@ More infos on Les Fruits Défendus
...
 
-Indices and tables
-------------------
+What's new in Saskatoon 2022
+----------------------------
+
+The collective worked hard over the last two years to develop an updated version of Saskatoon.
+We fixed many bugs and added a few improvements. 
+
+Changes that affects pick-leaders
```
Comment (2ynn) [inline docs/source/index.rst]: changes that *affect*

File: docs/source/index.rst [hunk 2]
```diff
@@ -33,8 +25,70 @@ More infos on Les Fruits Défendus
...
+
+   --
+   Saskatoon Harvest System
+   
+  Pick-leader still need to organize with selected pickers.
 
-* :ref:`genindex`
-* :ref:`search`
+
+Changes that mostly affects core members
```
Comment (2ynn) [inline docs/source/index.rst]: same here

File: docs/source/index.rst [hunk 3]
```diff
@@ -33,8 +25,70 @@ More infos on Les Fruits Défendus
...
+   --
+   Saskatoon Harvest System
+
+- The process to register new users has been enhanced such that an "Auth-User" is always created when using this new form: https://saskatoon.lesfruitsdefendus.org/person/create/.
+  You will be asked to select role(s) for this new user. The roles determine the privileges of the user (see https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/doc/permissions.org for more infos).
+  Then, use the "Register User" button from the community page to create a password for this user.
+
+- The process to validate new properties has been enhanced such that:
+  
+  - A list of similar properties is listed to help determine if the property already exist in database.
```
Comment (2ynn) [inline docs/source/index.rst:83]: NB: we should really test this and adjust filtering criteria, especially now that phone numbers are properly formatted using phone fields

**Review by 2ynn — APPROVED** [review]
thank you so much for this!

File: docs/source/index.rst (hunk 1 above)
Comment (tristanlatr) [inline docs/source/index.rst]:
```suggestion
Changes that affect pick-leaders
```

File: docs/source/index.rst (hunk 2 above)
Comment (tristanlatr) [inline docs/source/index.rst]:
```suggestion
Changes that mostly affect core members
```

File: docs/source/index.rst (hunk 3 above)
Comment (tristanlatr) [reply docs/source/index.rst:83]: Yes, this is why I've noted that core members should still double check manually. Proper tests are needed to determine in what extend we can trust this to uncover all cases of duplications.

**Pull request merged**